metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -7,14 +7,14 @@ package org.hypertrace.alert.engine.eventcondition.config.service.v1;
 message MetricAnomalyEventCondition {
   MetricSelection metric_selection = 1;
   repeated ViolationCondition violation_condition = 2;
+  string rule_duration = 3; // iso duration string
```

Comment:
shall we just keep `duration`?

Reply:
I somehow prefer the current name over `duration`